sdk/textanalytics/Azure.AI.TextAnalytics/tests/AnalyzeOperationTests.cs

```diff
@@ -0,0 +1,410 @@
+                }
+            };
+
+            AnalyzeOperationOptions operationOptions = new AnalyzeOperationOptions()
+            {
+                KeyPhrasesTaskParameters = new KeyPhrasesTaskParameters()
+                {
+                    ModelVersion = "latest"
+                },
+                EntitiesTaskParameters = new EntitiesTaskParameters()
+                {
+                    ModelVersion = "latest"
+                },
+                PiiTaskParameters = new PiiTaskParameters()
+                {
+                    ModelVersion = "latest"
+                },
+                DisplayName = "AnalyzeOperationWithMultipleTasks"
+            };
+
+            AnalyzeOperation operation = await client.StartAnalyzeOperationBatchAsync(batchDocuments, operationOptions);
+
+            await operation.WaitForCompletionAsync();
+
+            AnalyzeOperationResult resultCollection = operation.Value;
+
+            RecognizeEntitiesResultCollection entitiesResult = resultCollection.EntitiesResult;
+
+            ExtractKeyPhrasesResultCollection keyPhrasesResult = resultCollection.KeyPhraseResult;
+
+            RecognizePiiEntitiesResultCollection piiResult = resultCollection.PiiEntitiesResult;
+
+            Assert.IsNotNull(keyPhrasesResult);
+            Assert.IsNotNull(entitiesResult);
+            Assert.IsNotNull(piiResult);
+
+            // Keyphrases
+            Assert.AreEqual(2, keyPhrasesResult.Count);
+
+            var keyPhrasesListId1 = new List<string> { "Bill Gates", "Paul Allen", "Microsoft" };
+            var keyPhrasesListId2 = new List<string> { "gato", "perro", "veterinario" };
+
+            foreach (string keyphrase in keyPhrasesResult[0].KeyPhrases)
+            {
+                Assert.IsTrue(keyPhrasesListId1.Contains(keyphrase));
+            }
+
+            foreach (string keyphrase in keyPhrasesResult[1].KeyPhrases)
+            {
+                Assert.IsTrue(keyPhrasesListId2.Contains(keyphrase));
+            }
+
+            // Entities
+            Assert.AreEqual(2, entitiesResult.Count);
+
+            Assert.AreEqual(3, entitiesResult[0].Entities.Count);
+
+            var entitiesList = new List<string> { "Bill Gates", "Microsoft", "Paul Allen" };
+            foreach (CategorizedEntity entity in entitiesResult[0].Entities)
+            {
+                Assert.IsTrue(entitiesList.Contains(entity.Text));
+                Assert.IsNotNull(entity.Category);
+                Assert.IsNotNull(entity.Offset);
+                Assert.IsNotNull(entity.ConfidenceScore);
+            }
+
+            // PII
+
+            Assert.AreEqual(2, entitiesResult.Count);
+
+            Assert.AreEqual(3, entitiesResult[0].Entities.Count);
+            Assert.IsNotNull(entitiesResult[0].Id);
+            Assert.IsNotNull(entitiesResult[0].Entities);
+            Assert.IsNotNull(entitiesResult[0].Error);
+
+        }
+
+        [Test]
+        public async Task AnalyzeOperationWithSkipParameter()
```

Comment:
actually, because the whole test is "useless" until the `skip` works, you could write it as you will expect it to work but then ignore the whole test.
```
[Test]
[Ignore("reason...and link to issue")]
```

When the service provides the functionality it is easier to delete this than change the whole test

Reply:
that way you also don't have to save a recording that is not needed

Reply:
this should be fixed